### ISSUE TITLE
There is no image after connection

### ISSUE THREAD

**Issue by yolki4** [Hackinside/PS5_camera_files#18]
After carrying out all the installation points, the camera is detected but does not display an image anywhere, I tried several times to reinstall everything according to the instructions, but there is no result, in applications the camera is detected but does not give an image
![Screenshot_1](https://github.com/Hackinside/PS5_camera_files/assets/162746978/3178e5e8-623e-447b-a0c8-c3c821839999)

**Comment by jmulino** [comment]
I have the exact same problem, are you on windows 11?

**Comment by Daemon130** [comment]
it does the same to everyone, i wanted to contact the man who did that the program but i don't think i can dm him

**Comment by ligeonpro19** [comment]
Еxactly the same situation. According to my observations, this happens if the system is based on intel, but regardless of the system version.

**Comment by shinobi2724** [comment]
У меня ryzen и windows 10,но все равно не показывает изображение((

**Comment by Daemon130** [comment]
we can dm him by sending him a tip but i don't really want to use money to make this camera works

**Comment by Tubelaxo** [comment]
I think it is the problem with the drivers since they were 3 years ago and things change. I have the same error :/.

**Comment by MisereMortem** [comment]
Yeah, it doesn't work for me either. The old PS4 camera also doesn't work anymore, though it kinda worked before. And I bought PS5 camera specifically to use on my PC. Well, I bought it new for only like 6$, and I also have PS5, but I don't really need it there.
